simple/sample/output/process/debug_resolve_powerplants.csv

```diff
@@ -1,6 +1,6 @@
 input,dcid,link
-FOO BAR,*UNRESOLVED*,
```

Comment:
this file seems to change every time, is there a way to make it stable?

Reply:
Yeah it's annoying. Will look into it after the next PR.